src/app/world-map/world-map.component.ts

```diff
@@ -0,0 +1,34 @@
+import { AfterViewInit, Component } from '@angular/core';
+import * as L from 'leaflet';
+
+@Component({
+  selector: 'app-world-map',
+  templateUrl: './world-map.component.html',
+  styleUrls: ['./world-map.component.scss'],
+})
+export class WorldMapComponent implements AfterViewInit {
+  private map;
+
+  constructor() {}
+
+  ngAfterViewInit(): void {
+    this.initMap();
+  }
+
+  private initMap(): void {
+    this.map = L.map('map', {
+      center: [51.0, 0.0],
+      zoom: 3,
```

Comment:
Usually a good idea to pull out the constants to some more visible spot. Call it e.g. `DEFAULT_MAP_POSITION`.
Another option would be to pass the latitude and longitude in a constructor and/or have a setter for them.

The functionality I'm thinking about is user choosing a country from a drop-down list and the map automatically centering on the country. I don't know how leaflet handles redrawing maps, so not sure about the best implementation at the moment.